### PR TITLE
Replaced the LC with the TSC.

### PR DESCRIPTION
**Pull request by dblock**
### Description

The OpenSearch Project is supported by [The OpenSearch Software Foundation](https://foundation.opensearch.org/), a project of [The Linux Foundation](https://www.linuxfoundation.org/). You can read the launch announcement [here](https://www.linuxfoundation.org/press/linux-foundation-announces-opensearch-software-foundation-to-foster-open-collaboration-in-search-and-analytics) and learn more about joining the foundation [here](https://foundation.opensearch.org/). 

The project has a [Governing Board](governing-board/README.md) and a [Technical Steering Committee](technical-steering-committee/README.md).

This PR adds the initial TSC charter and creates a placeholder for the GB. 

Closes https://github.com/opensearch-project/community/issues/48. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by krisfreedain** [comment]
thank you @dblock - one comment - completely erasing the records of the LC feels inappropriate for transparency - I'd say we should likely note that it is super-ceded by TSC at least for a set time-period

**Comment by dblock** [comment]
> thank you @dblock - one comment - completely erasing the records of the LC feels inappropriate for transparency - I'd say we should likely note that it is super-ceded by TSC at least for a set time-period

I replaced MEMBERS.md in the LC folder by a note with a link to the last working version with the list of people and added a README with the same to access meeting notes. Does that work?

**Review by andrross — COMMENTED** [review]
Looks like a `.DS_Store` file slipped in here

**Review by KarstenSchnitter — APPROVED** [review]
Good work. Only minor remarks.

File: technical-steering-committee/CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+      9. coordinating any marketing, events, or communications regarding the Project.
+
+3. **TSC Voting**
+
+   1. While the Project aims to operate as a consensus-based community, if any TSC decision requires a vote to move the Project forward, the voting members of the TSC will vote on a one vote per voting member basis.
+
+   2. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
+
+   3. Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
```
[the remark this answers is not in the document]
Comment (dblock) [reply technical-steering-committee/CHARTER.md]: Good catch, I lost that in some export / import. I believe this is correct now, check again?

**Review by KarstenSchnitter — APPROVED** [review]

**Review by AmiStrn — APPROVED** [review]
Nothing to add. This is great, Thanks @dblock !

File: technical-steering-committee/CHARTER.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+   b. The scope of the Project includes collaborative development under the Project License (as defined herein) supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open-source project.
+
+2. **Technical Steering Committee**
+
+   a. The Technical Steering Committee (the "TSC") will be responsible for all technical oversight of the open-source Project. 
+
+   b. Startup Period.
+
+      1. The initial voting members ("Initial Members") of the TSC at the inception of the Project (the "Start Date") are those individuals identified as voting members in the "STEERING.md" file within the Project’s .github repository. Initial Members will consist of Initial Members serving one-year terms and Initial Members serving two-year terms, as indicated in the STEERING.md file.
```
Comment (reta) [inline technical-steering-committee/CHARTER.md:21]: I believe we set on TSC to be one year term?

**Review by reta — APPROVED** [review]
Thanks @dblock !

File: technical-steering-committee/CHARTER.md (hunk 2 above)
Comment (dblock) [reply technical-steering-committee/CHARTER.md:21]: We always need some continuity, so it's 2 years to create overlap. At start LC members are 1 year so we can re-elect half after year 1, and the rest 2 years so we can re-elect the other half and then it's business as usual.

**Review by reta — APPROVED** [review]

**Review by samuel-oci — APPROVED** [review]

**Comment by samuel-oci** [comment]
Thank you @dblock !

**Comment by dblock** [comment]
@krisfreedain Can this be merged?

File: technical-steering-committee/CHARTER.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+      5. appointing representatives to work with other open source or open standards communities;
+
+      6. establishing community norms, workflows, issuing releases, security issue reporting policies and the Elections Procedures Document (defined below);
+
+      7. approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the series manager of the Project (as provided for in the Series Agreement, the "Series Manager") to resolve matters or concerns that may arise as set forth in Section 7 of this Charter;
+
+      8. discussions, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
+
+      9. coordinating any marketing, events, or communications regarding the Project.
```
Comment (rbowen) [inline technical-steering-committee/CHARTER.md:63]: This seems wrong. That's *definitely* not the job of the TSC. Point 9 is the job description of the community manager, and would be a significant distraction from the TSC's correct focus - on the steering the technical direction of the project.

Comment (dblock) [reply technical-steering-committee/CHARTER.md:63]: Thanks @rbowen.

I think this item came from the template we used and is present in other charters under LF, [here](https://openmessaging.cloud/charter/) and [here](https://caseontology.org/resources/charter.html). 

Removing that paragraph would be an amendment to the Technical Charter that was reviewed by the members before we announced the foundation, which requires a 2/3 vote of the entire TSC and approval by LF. I opened https://github.com/opensearch-project/community/issues/53, please add your comments.

**Comment by dblock** [comment]
We had a discussion on whether this content belongs in the community repo and decided to create a new **steering** one. @krisfreedain is taking care of it.

I'll split this PR in two then.

**Comment by dblock** [comment]
I've moved the TSC parts to https://github.com/opensearch-project/technical-steering (members merged as part of https://github.com/opensearch-project/technical-steering/pull/4).

@krisfreedain take a look again?

**Pull request merged**